http/src/main/resources/reference.conf

```diff
@@ -603,13 +603,17 @@ gke {
     dbPassword = "replace-me"
     enabled = true
   }
-  rstudioApp {
+  aouApp {
     # If you update this here, be sure to update in the dockerfile
-    chartName = "/leonardo/aou-rstudio-chart"
-    chartVersion = "0.1.0"
+    # This is really just a prefix of the chart name. Full chart name for AOU app will be
+    # this chartName in config file appended with allowedChartName from createAppRequest
```

Comment:
The caller of Leo then needs to know the exact chart name? 

Think about injection attacks, if we're taking user input and appending to file system paths.

(Thinking what if I passed `"allowedChartName": "aou-sas; rm -rf /"` heh)

Reply:
so the decoder will reject this. Btw, the chart name will be the fullname is what I'm currently thinking like (aou-sas-chart)...

```
  implicit val allowedChartNameDecoder: Decoder[AllowedChartName] =
    Decoder.decodeString.emap(x => AllowedChartName.stringToObject.get(x).toRight("chart name not allowed"))
```